src/app/components/kit/showKit/showKit.html

```diff
@@ -118,12 +118,21 @@ <h2 class="title">
             </div>
             <div class="sensor_select">
               <!--<md-icon md-svg-src="{{ vm.selectedSensorData.icon }}" class="sensor_icon_selected"></md-icon>-->
+<<<<<<< HEAD
```

Comment:
@tuengominh did you use `git add -p`?

Reply:
Yes I did, but I didn't see any message about conflict... But I also noticed that this pull request included commits from my previous pull request @-@. Lemme research about it, revert some commits and also add the SD card button to user profile view...